debian/control

```diff
@@ -15,7 +15,7 @@ Multi-Arch: same
 Depends: lcui (= ${binary:Version}), ${misc:Depends}
 Description: A small C library for building user interfaces.
 
-Package: lcui
+Package: lcui1
```

Comment:
I found this document: https://www.netfort.gr.jp/~dancer/column/libpkg-guide/libpkg-guide.html#naminglibpkg

We should rename the package to `liblcui1`